common/metrics/src/lib.rs

```diff
@@ -76,6 +77,12 @@ lazy_static! {
     .unwrap();
     static ref K8S_LINES: IntCounter =
         register_int_counter!("logdna_agent_k8s_lines", "Kubernetes event lines read").unwrap();
+    static ref K8S_NAMESPACE_LINES: IntCounterVec = register_int_counter_vec!(
```

Comment:
I wish we could just conditionally add a label if the K8S stuff is on, but alas, they're using `lazy_static!` so it might not be worth the effort to plumb it through. :(

Reply:
I think I see what you are saying. It's might make sense to alter the FS_LINES to a CounterVec and change the increment function to accept an optional namespace argument. If it's not there, either emit a blank label (unsure of the semantics) or a default label (something like "sys" or "all" or "global")